app/controllers/games_controller.rb

```diff
@@ -0,0 +1,28 @@
+class GamesController < ApplicationController
+	# before_action :authenticate_user!
+  
+  def show
+  	@game = Game.find(params[:id])
+  end
+
+  def new
+  	@game = Game.new
+  end
+
+  def create
+  	@game = Game.new(game_params)
+  	if @game.save
+  		redirect_to game_path(@game)
+  	else
+  		render :new, :status => :unprocessable_entity 
+  	end
+
+  end
+
+  private
+
+  def game_params
+		params.require(:game).permit(:name)  	
+  end
+
+end
```

Comment:
@KaoruDev My sublime default spacing is "Tab Size: 2" is there an easy way to change the default to Spaces: 2 for all my projects and have it automatically apply when I open a file?